tests/PHPUnit/Framework/TestCase/UnitTestCase.php

```diff
@@ -18,16 +21,45 @@
  */
 abstract class UnitTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
@tsteurs in that case we fix it: we move tests that are not unit tests into integration tests or system tests. Once that is done, we deprecate the class, and later we can safely remove it.